src/System.Runtime.InteropServices.RuntimeInformation/src/System.Runtime.InteropServices.RuntimeInformation.builds

```diff
@@ -15,13 +15,22 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Runtime.InteropServices.RuntimeInformation.csproj">
-      <TargetGroup>net46</TargetGroup>
+      <OSGroup>Windows_NT</OSGroup>
```

Comment:
Desktop facades are special.  They must be passed to the compiler to unify with classic mscorlib-based reference assemblies.  If we didn't do that desktop would get the reference assemblies and complain that types exist in both assemblies A and B.  On second thought, double check that this is actually a facade, if so it needs to go in the reference package.  If not it can go in the windows runtime package.